src/pages/atomicdex/api/v20/index.mdx

```diff
@@ -214,17 +205,220 @@ The `EvmNode` object includes the following items for a given coin or token:
+| to_funding_value            | integer | Optional. The maximum value of channel funding in satoshis. PARAM NAME NOT CONSISTENT.     |
+| channel_type                | string  | Optional. `Inbound` or `Outbound`. INCONSISTENT WITH `is_outbound` PARAM  |
+| closing_tx                  | integer | Optional. A transaction ID which closed the channel.    |
+| closure_reason              | integer | Optional. The reason a channel was closed.   |
+| claiming_tx                 | integer | Optional. A transaction ID which claimed funds from the channel.    |
+| from_claimed_balance        | integer | Optional. The minimum balance of channel funds claimed in satoshis.    |
+| to_claimed_balance          | integer | Optional. The maximum balance of channel funds claimed in satoshis.    |
+| channel_visibility          | integer | Optional. `Public` or `Private`.   |
+
+
+### LightningOpenChannelsFilter
+
+| Parameter                   | Type    | Description                                                                                                     |
+|-----------------------------|---------|-----------------------------------------------------------------------------------------------------------------|
+| channel_id                  | string  | Optional. Unique string identifying a channel by its ID.                                                        |
+| counterparty_node_id        | string  | Optional. A hexidecimal string identifying a counterparty node. |
+| funding_tx                  | string  | Optional. A transaction ID which added funds.                   |
+| from_funding_value_sats     | integer | Optional. The minimum value of channel funding in satoshis.     |
+| to_funding_value_sats       | integer | Optional. The maximum value of channel funding in satoshis.     |
+| is_outbound                 | boolean | Optional. If `true`, limits the response to outbound channels only. |
+| from_balance_msat           | integer | Optional. The minimum channel balance in millisatoshis.     |
+| to_balance_msat             | integer | Optional. The maximum channel balance in millisatoshis.     |
+| from_outbound_capacity_msat | integer | Optional. The minimum outbound capacity of the channel balance in millisatoshis.  |
+| to_outbound_capacity_msat   | integer | Optional. The maximum outbound capacity of the channel balance in millisatoshis.  |
+| from_inbound_capacity_msat  | integer | Optional. The minimum inbound capacity of the channel balance in millisatoshis.   |
+| to_inbound_capacity_msat    | integer | Optional. The maximum inbound capacity of the channel balance in millisatoshis.   |
+| confirmed                   | boolean | Optional. If `true`, channel is confirmed NEED MORE INFO |
+| is_usable                   | boolean | Optional. If `true`, channel is usable NEED MORE INFO    |
+| is_public                   | boolean | Optional. If `true`, channel is public NEED MORE INFO    |
```

Comment:
confirmed: If `true`, limits the response to only channels with channel opening transactions that passed the number of confirmations required for the channel to be usable.

is_usable: If `true`, limits the response to only channels that are confirmed and the counterparty is online, meaning that these channels can be used for payments.

is_public: If `true`, limits the response to only channels that our node announces to the lightning network, these channels are visible on lightning explorers.

Reply:
thanks, fixed